lib/connections/nodemailer.js

```diff
@@ -8,21 +8,32 @@ const smtpConfig = {
   port: process.env.SMTP_PORT,
   secure: boolean(defaultTo(process.env.SMTP_SECURED, false)),
   ignoreTLS: boolean(defaultTo(process.env.SMTP_IGNORE_TLS, false)),
-  requireTLS: boolean(defaultTo(process.env.SMTP_REQUIRE_TLS, false)),
-  auth: {
-    user: process.env.SMTP_USER,
-    pass: process.env.SMTP_PASS
-  }
+  requireTLS: boolean(defaultTo(process.env.SMTP_REQUIRE_TLS, false))
 };
 
+// Placed like this since the boolean set default can be hard to read in an if condition.
+const sendAuthRequest = boolean(defaultTo(process.env.SMTP_AUTH_REQ, true));
+
+//Do not want the auth object in the smtpConfig object if sendAuthRequest is false, default is true.
```

Comment:
Comment should have consistent spacing

```suggestion
// Do not want the auth object in the smtpConfig object if sendAuthRequest is false, default is true.
```